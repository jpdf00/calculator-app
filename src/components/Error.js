import React from 'react';

const Error = () => (
  <div>
    <h1>404 - Page not found!</h1>
  </div>
);

export default Error;
