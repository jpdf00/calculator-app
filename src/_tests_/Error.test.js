import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Error from '../components/Error';

describe('Error Component', () => {
  test('It should render', () => {
    const error = renderer.create(
      <BrowserRouter>
        <Error />
      </BrowserRouter>,
    ).toJSON();
    expect(error).toMatchSnapshot();
  });
});
