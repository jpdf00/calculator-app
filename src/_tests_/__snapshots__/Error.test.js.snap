// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component It should render 1`] = `
<div>
  <h1>
    404 - Page not found!
  </h1>
</div>
`;
